src/usbhost/pipe-usbhost.cpp

```diff
@@ -0,0 +1,61 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+
+#include "usbhost.h"
+#include "pipe-usbhost.h"
+#include "../types.h"
+
+namespace librealsense
+{
+    namespace platform {
+        pipe::pipe(::usb_device* handle) :
+                _handle(handle),
+                _pull_thread(nullptr),
+                _pull_requests(true)
+        {
+            start();
+        }
+
+        pipe::~pipe()
+        {
+            stop();
+        }
+
+        void pipe::start()
+        {
+            _pull_requests = true;
```

Comment:
Imo start/stop should be mutex-protected

Reply:
only called on constructor / destructor, function will be removed